docs/slack-connector/installation.md

```diff
@@ -18,29 +18,35 @@ The Slack Connector is a component which allows interaction with the Slack API f
 
 ### Prerequisites
 
-- Slack Bot with desired privileges installed to the destination workspace. See the tutorial provided by Slack on how to setup an application [here](https://api.slack.com/bot-users#getting-started). After you have created the Signing Secret, it appears in the application's **Settings** in the **Basic Information** section.
-- Access to Kyma Console
+- Slack App with desired privileges installed to the destination workspace. See the tutorial provided by Slack on how to setup an application [here](https://api.slack.com/bot-users#getting-started).
 
-> **OPTIONAL:** Follow these steps to install the default application. Be aware that it has **full permissions** in the workspace.
->
-> 1. Go to the [authentication page](https://auth-slack.herokuapp.com/). Click the **Add to Slack** button, which redirects you to another page. Select the desired workspace and click **Allow**.
->       - **NOTE:** If the link does not work, see [this](https://api.slack.com/docs/oauth#flow) tutorial in the Slack API documentation to create your own application.
-> 2. Copy the Bot Authentication Token and/or Slack Signing Secret. You will need it later in the installation process.
+    After creating the app:
+    - The Slack Application's {SIGNING_SECRET} is used for validating requests coming from Slack by verifying its unique signature. Find it in the [Application's](https://api.slack.com/apps) **Settings** in the **Basic Information** section. [Learn more](https://api.slack.com/docs/verifying-requests-from-slack).
+    - The Slack Application's {SLACK_TOKEN} is used for requests authorization. Find it in the [Application's](https://api.slack.com/apps) **Features** in the **OAuth & Permissions** section. Depending on the usecase it will be *OAuth Access Token* or *Bot User OAuth Access Token*. [Learn more](https://api.slack.com/docs/oauth).
```

Comment:
```suggestion
    - The Slack Application's {SLACK_TOKEN} is used for requests authorization. Find it in the [Application's](https://api.slack.com/apps) **Features** in the **OAuth & Permissions** section. Depending on the usecase it will be **OAuth Access Token** or **Bot User OAuth Access Token**. [Learn more](https://api.slack.com/docs/oauth).
```